site/sass/components/_nav.scss

```diff
@@ -257,7 +238,7 @@
 
   &__primary-item {
     @include minLangMQ($bkpt-map--header--large) {
-      padding-bottom: 1rem;
+      padding-bottom: 1.18rem;
```

Comment:
This number seems off to me; it doesn't align to a `px` value (it's `18.88px`) and it doesn't align to other scales we have. Where did it come from? Can we use `1.25rem` instead?

Reply:
It actually comes from `19px` but yes I can change this value to `1.25rem`

Reply:
Fixed.